src/inverseRobot.js

```diff
@@ -7,7 +8,27 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  for (const key in robot) {
+    const property = robot[key];
+    let count = 0;
+
+    for (const key2 in robot) {
```

Comment:
circle inside another circle it isn't good practice, try to init object and in first circle check ```
yourVariable.hasOwnProperty(robot[key]) {
      return null;
}
if not yourVariable[robot[key]] = key;
```